tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature

```diff
@@ -69,6 +69,7 @@ Feature: Sharing files and folders with internal groups
     When the user renames file "lorem.txt" to "new-lorem.txt" using the webUI
     And the user shares file "new-lorem.txt" with group "grp1" as "Editor" using the webUI
     And the user re-logs in as "user1" using the webUI
+    And the user browses to the files page
```

Comment:
can we call `waitTillThumbnailsLoaded` function in the `user has logged in` or `user has re-logged in` step. 
That way we can avoid having `user has browsed to the files page` everywhere after logging in.